docs/sources/release-notes/v2.3.md

```diff
@@ -29,6 +29,11 @@ For the complete list of changes, see the [Changelog](https://github.com/grafana
 - **Added `backfill` command to mimirtool to upload Prometheus TSDB blocks:**
   The `mimirtool backfill` command will make the process of back-filling Prometheus TSDB blocks, using an API available in the compactor, even easier which allows for a migration to Grafana Mimir while keeping your previous Prometheus TSDB data.
 
+- **Helm Helm Chart improvements:** The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.3 release, we’re also releasing version 3.1 of the Helm chart. Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+  - The MinIO sub chart dependency is upgraded from a deprecated Helm chart (https://helm.min.io/) to the supported chart's version (https://charts.min.io/). This includes a **breaking change** of how the administrator password is set, however as the _built-in_ MinIO is not recommended for production, this change did not warrant a new major version of the Mimir Helm chart.
+  - The backfill API endpoints are now exposed on the Nginx gateway.
+  - Default to injecting the `no_auth_tenant` from the Mimir configuration as the value for `X-Scope-OrgID` in Nginx instead of hard coded `anonymous`.
```

Comment:
I would clarify that the default behaviour hasn't changed tho.

Reply:
tried to clarify in a suggestion bellow, wdty? @pracucci @09jvilla